packages/mirinae/src/inputs/buttons/toggle-button/PToggleButton.stories.mdx

```diff
@@ -59,18 +62,95 @@ export const Template = (args, { argTypes }) => ({
     </Story>
 </Canvas>
 
-## Label
+## ShowStateText
 <Canvas>
-    <Story name="toggle button label">
+    <Story name="show toggle button state text">
```

Comment:
suggestion: Story name is shown in the side bar. It's kind of the title.

e.g.
![image](https://github.com/cloudforet-io/console/assets/26986739/9d688f40-05e4-4faf-ad89-11ee65e854f4)

So I recommend "Show Text Next to Toggle Button"